src/CalendarBody.tsx

```diff
@@ -189,7 +189,7 @@ export function _CalendarBody({
             ))}
             {dayJsConvertedEvents
               .filter(({ start }) =>
-                start.isBetween(date.startOf('day'), date.endOf('day'), null, '[)'),
+                (start as dayjs.Dayjs).isBetween(date.startOf('day'), date.endOf('day'), null, '[)'),
```

Comment:
Do you have any way to omit the castings `as dayjs.Dayjs`?
It should be more readable.